python/ribasim/ribasim/model.py

```diff
@@ -738,3 +741,67 @@ def _add_allocation(self, uds):
             uds[varname] = da
 
         return uds
+
+    def to_fews(
+        self,
+        region_home: str | PathLike[str],
+        network: bool = True,
+        results: bool = True,
+    ) -> None:
+        """
+        Write the model network and results into files used by Delft-FEWS.
+
+        ** Warning: This method is experimental and is likely to change. **
+
+        To run this method, the model needs to be written to disk, and have results.
+        The Node and Link tables are written to shapefiles in the Config directory.
+        The results are written to NetCDF files in the Modules directory.
+        The netCDF files are NetCDF4 with CF-conventions.
+
+        Parameters
+        ----------
+        region_home: str | PathLike[str]
+            Path to the Delft-FEWS REGION_HOME directory.
+        """
+        region_home = DirectoryPath(region_home)
+        if network:
+            self._network_to_fews(region_home)
+        if results:
+            self._results_to_fews(region_home)
+
+    def _network_to_fews(self, region_home: DirectoryPath) -> None:
+        """Write the Node and Link tables to shapefiles for use in Delft-FEWS."""
+        df_link = self.link.df
+        df_node = self.node_table().df
+        assert df_link is not None
+        assert df_node is not None
+
+        network_dir = region_home / "Config/MapLayerFiles/{ModelId}"
+        link_path = network_dir / "{ModelId}Links.shp"
+        node_path = network_dir / "{ModelId}Nodes.shp"
```

Comment:
Did you mean these to be f-strings? If so, you miss the f, and the argument. edit: Maybe not, given the rest of the code, what a weird way of naming files.

Reply:
Yeah it is a bit odd, the idea was that this way it can easily be replaced. We want to support importing multiple different models in the future.